packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -83,12 +86,12 @@ const BrandMessaging = ({
               </div>
             </div>
             <div className="flex-initial max-w-md pr-2">
-              <p className="text-left text-sm font-bold">
+              <p className="text-left text-sm font-bold w-5/6 mobileLandscape:w-full">
                 <FtlMsg id="brand-prelaunch-title">
                   Firefox accounts will be renamed Mozilla accounts on Nov 1
                 </FtlMsg>
               </p>
-              <p className="text-left text-xs">
+              <p className="text-left text-xs w-5/6 mobileLandscape:w-full">
```

Comment:
`text-left` should probably be replaced with `text-start` (this will left-align when direction is ltr and right align when direction is rtl)

Reply:
Now using test-start